docs/tutorials/23.2-monorepo-rush.md

```diff
@@ -36,14 +36,14 @@ At this point we should see four items in the directory:
-5. *.travis.yml* (from the second command, configures the [Travice CI](https://www.travis-ci.com/) service)
-6. *common* (from the second command, rush stores all files it needs in there. Read more about this in rush's [Config file reference](https://rushjs.io/pages/advanced/config_files/))
+5. *.travis.yml* (from the second command, configures the [Travis CI](https://www.travis-ci.com/) service)
+6. *common* (from the second command, Rush stores all files it needs in there. Read more about this in Rush's [config file reference](https://rushjs.io/pages/advanced/config_files/))
 
-Great! Now we need to configure rush by opening rush.json and performing the following steps:
+Great! Now we need to configure rush by opening *rush.json* and performing the following steps:
 
-- **Choose a package manager:** Search for `npmVersion` inside the rush.json file. You'll see that rush uses pnpm as a default. Since i only tried this with npm we uncomment the `npmVersion` line and comment the `pnpmVersion` line. You should also see if you have the latest npm version installed globally and set this version here. You can of course feel free to use your preferred package manager. Read more about those [here](https://rushjs.io/pages/maintainer/package_managers/).
+- **Choose a package manager:** Search for `npmVersion` inside the *rush.json* file. You'll see that rush uses pnpm as a default. Since we only tried this with npm we uncomment the `npmVersion` line and comment the `pnpmVersion` line. You should also see if you have the latest npm version installed globally and set this version here. You can of course feel free to use your preferred package manager. Read more about those [here](https://rushjs.io/pages/maintainer/package_managers/).
 - **Custom npm registry:** If you use a custom npm registry you might configure it in `common\config\rush\.npmrc`. Read more about this [here](https://rushjs.io/pages/maintainer/npm_registry_auth/).
-- **Folder structure:** Since i don't want to mix rush's files with my packages i add a folder "packages" in the root directory by running `mkdir tools`. All pilets and the app shell will live in their own folders directly inside the packages folder. In this scenario they are nested in a folder depth of 2 which is rush's default. If you want do have a different structure you might need to configure `projectFolderMinDepth` and `projectFolderMaxDepth` inside rush.json. I'll just uncomment them being set to 2.
+- **Folder structure:** Since we don't want to mix rush's files with my packages we add a folder "packages" in the root directory by running `mkdir tools`. All pilets and the app shell will live in their own folders directly inside the packages folder. In this scenario they are nested in a folder depth of 2 which is Rush's default. If you want do have a different structure you might need to configure `projectFolderMinDepth` and `projectFolderMaxDepth` inside *rush.json*. I'll just uncomment them being set to 2.
```

Comment:
```suggestion
- **Folder structure:** Since we don't want to mix Rush's files with our packages, we add a folder "packages" in the root directory by running `mkdir tools`. All pilets and the app shell will live in their own folders directly inside the packages folder. In this scenario they are nested in a folder depth of 2 which is Rush's default. If you want to have a different folder structure you might need to configure `projectFolderMinDepth` and `projectFolderMaxDepth` inside *rush.json*. I'll just uncomment them being set to 2.
```

Reply:
Actually, quick question. The doc says that we create a folder "packages", but the command says "mkdir tools", i.e. not "packages". Is that a mistake?